src/printer.js

```diff
@@ -878,11 +878,13 @@ function genericPrintNoParens(path, options, print, args) {
     case "ThisExpression":
       return "this";
     case "Super":
-      return "super"; // Babel 6 Literal split
+      return "super";
+    // Babel 6 Literal split
```

Comment:
Should this be above `"super"`? That's what I'd think looking at the old version